cxx-extras-Alpine/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine
```

Comment:
The CI builds for the target `deploy`, so you will need to add it to all the `Dockerfile`s.

```suggestion
FROM alpine AS deploy
```

Reply:
Should be fixed now